dev/change-version-to-2.11.sh

```diff
@@ -17,10 +17,4 @@
 # limitations under the License.
 #
 
-# Note that this will not necessarily work as intended with non-GNU sed (e.g. OS X)
-BASEDIR=$(dirname $0)/..
-find $BASEDIR -name 'pom.xml' | grep -v target \
-  | xargs -I {} sed -i -e 's/\(artifactId.*\)_2.10/\1_2.11/g' {}
-
-# Also update <scala.binary.version> in parent POM
-sed -i -e '0,/<scala\.binary\.version>2.10</s//<scala.binary.version>2.11</' $BASEDIR/pom.xml
+$(dirname $0)/change-scala-version.sh 2.10 2.11
```

Comment:
@ScrapCodes should we just delete these two scripts and update any references to use the parameterized, new script? seems cleaner.

LGTM in any event.